src/System.Memory/src/System/Buffers/Text/Utf8Formatter/Utf8Formatter.Integer.Signed.Default.cs

```diff
@@ -48,17 +48,6 @@ private static bool TryFormatInt64Default(long value, Span<byte> destination, ou
             }
         }
 
-        // TODO: Use this instead of TryFormatInt64Default to format numbers less than int.MaxValue
```

Comment:
@ahsonkhan, is this Todo still meaningful? Is there an issue for it?

Reply:
It might lead to perf optimization for x86 but can be removed for now since it requires an investigation. Filed an issue instead: https://github.com/dotnet/corefx/issues/32703

Reply:
It might lead to perf optimization for x86 but can be removed for now since it requires an investigation. Filed an issue instead: https://github.com/dotnet/corefx/issues/32703